src/index.js

```diff
@@ -1580,10 +1583,11 @@ module.exports = {
 			this.settings.routes.forEach(route => this.addRoute(route));
 
 		// Regenerate all auto aliases routes
+		const debounceTime = (debounceTime => (!isNaN(debounceTime) && debounceTime >= 0) ? debounceTime : 500)(parseInt(this.settings.debounceTime));
```

Comment:
I feel it's a little too complicated. Just use a simply:
`const debounceTime = debounceTime > 0 ? Number(this.settings.debounceTime) : 500`

Reply:
I mean
1. if `this.settings.debounceTime` is not set, `parseInt` will be NaN
2. if `this.settings.debounceTime` is set, but
- set to a wrong value likes 'a', `parseInt` will be NaN
- `debounceTime` can not less than 0 too
3. If pass all conditions, take return value from `parseInt`, else take default `500`

This line is short for
```js
function parseDebounceTime(settingValue) {
  const tmp = parseInt(settingValue);
  if (!isNaN(tmp) && tmp > 0) return tmp;
  else return 500
}
const debounceTime = parseDebounceTime(this.settings.debounceTime)
```

Reply:
I mean, but the logic is too complex. The option will be set by developers, and I don't want to protect the developers from themselves. If somebody set "a" for a `debounceTime` option....

Reply:
@icebob backlog idea: maybe the fastest-validator would help here to check the configuration parameters?